app/assets/javascripts/no_sunet.js

```diff
@@ -0,0 +1,43 @@
+$(document).on('click', '#no-sunet-button', function(){
+  setTimeout(function(){
+      $('#request_user_attributes_library_id').focus();
+  }, 1);
+
+  $('form').on('keypress', function(e) {
+    if(e.which == 13) {
+      $('.form-group input').checkForm(e);
+    }
+  });
+
+  $('.form-control').on('mouseenter mouseleave focus blur',function(e){
+    $('.form-group input').checkForm(e);
+  });
+
+  $('#no-sunet-submit').click(function(e) {
```

Comment:
:+1: @atz 

And for reference checkout the jquery [submit()](https://api.jquery.com/submit/) documentation.  See the `event.preventDefault()` in the example.